src/java/edu/ucsd/library/dams/api/DAMSAPIServlet.java

```diff
@@ -3375,6 +3306,23 @@ private Map mintDOI( String objid, TripleStore ts, TripleStore es, FileStore fs,
 			xml = obj.getRDFXML(true);
 		}
 
+		// pre-validate
```

Comment:
That's probably a good idea -- I could move the validation code into the Ezid utility, which makes sense from a domain point-of-view.